pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -147,12 +148,14 @@ class GatewayDataForm extends React.Component {
           max={8}
           placeholder={sharedMessages.gatewayEUI}
           component={Input}
+          glossaryId={glossaryIds.GATEWAY_EUI}
         />
         <Form.Field
           title={sharedMessages.gatewayName}
           placeholder={sharedMessages.gatewayNamePlaceholder}
           name="name"
           component={Input}
+          glossaryId={glossaryIds.GATEWAY_NAME}
```

Comment:
Do we need a tooltip for the `name` field? cc @kschiffer @benolayinka

Reply:
Yes, let's add one. We do the same for end devices as well.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/issues/3809#issuecomment-829087396